frontend/server/src/main/java/org/pytorch/serve/http/ExtendedSSLHandler.java

```diff
@@ -0,0 +1,41 @@
+package org.pytorch.serve.http;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.netty.handler.ssl.OptionalSslHandler;
+import io.netty.handler.ssl.SslContext;
+import io.netty.handler.ssl.SslHandler;
+import java.util.List;
+import org.pytorch.serve.util.ConfigManager;
+import org.pytorch.serve.util.NettyUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ExtendedSSLHandler extends OptionalSslHandler {
+    private static final Logger logger = LoggerFactory.getLogger(ExtendedSSLHandler.class);
+    /** the length of the ssl record header (in bytes) */
+    private static final int SSL_RECORD_HEADER_LENGTH = 5;
```

Comment:
Use `io.netty.handler.ssl.SslUtils.SSL_RECORD_HEADER_LENGTH` instead

Reply:
Tried that earlier however it is not visible[protected] netty.